sklearn/metrics/ranking.py

```diff
@@ -149,14 +141,32 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    interpolation : None (default), or string ['eleven_point']
+
+        ``None``:
+            Do not interpolate the average precision. Instead, compute
+            sum(p[i] * (r[i] - r[i-1])) for all precision, recall pairs
+            p[i], r[i] for i ≥ 1. This is the primary definition used in
+            the `Wikipedia entry for the Average precision
+            <http://en.wikipedia.org/wiki/Average_precision>`
+        ``'eleven_point'``:
+            For each of the recall values, r in {0, 0.1, 0.2, ..., 1.0},
+            compute the arithmetic mean of the first precision value with a
+            corresponding recall ≥ r. See the `Stanford Information Retrieval
```

Comment:
the ``>=`` symbol is probably not a good idea here (you could declare an encoding for the file but I think just writing >= is better).